### ISSUE TITLE
Delay when opening any container or inventory

### ISSUE THREAD

**Issue by MicHarKris** [Odie/skyui-vr#39]
Whenever I access a container (Dead guy, unlocked chest, etc) I experience a solid couple of seconds delay before the inventory opens.

**Comment by Odie** [comment]
This likely means the pex files that comes with SKSE we’re not installed/activated correctly.

**Comment by MicHarKris** [comment]
Thank you for the VERY IMPRESSIVELY QUICK REPLY! WOW! But I suspect that what actually made the difference was that the mod wasn't fully loaded into the game - I thought that all mods were loaded as you opened the game, but then suddenly I saw a couple of notifications arriving while playing that this and that mod had been activated. If not, the things I did before the problem became solved included the following:
- Load both SKSEVR and SKYUIVR into Vortex on top of the regular placement in the folders.
- Create a SKSE.INI file in the Data\SKSE folder.
- Be patient and wait for the mods to load in.
